rclpy/src/rclpy/node.hpp

```diff
@@ -42,6 +42,33 @@ get_node_name(py::capsule pynode);
  */
 const char *
 get_node_namespace(py::capsule pynode);
+
+/// Count publishers for a topic.
```

Comment:
@ahcorde meta: I think it'd be worth mentioning that this function returns the count of all the publishers known for that topic in the entire ROS graph.

Reply:
And replicate that extra information in the docstring.